## Nanogenmo Assignment for Computational Approaches to Narrative Fall 2019 ITP


#### The following code is exerpts taken from Allison Parrish's "Enough Python" and "Corpus Driven Narrative Generation" exercises in python.<br/> Please find her original code here: https://gist.github.com/aparrish<br/> Class syllabus can be found here:http://catn.decontextualize.com/

For our computer generated novel assingment for Allison Parrish's class, I generated text by running through certain personal images in Runway(https://runwayml.com/) through the image-to-text model. I wanted to make sense of the text in python and then reorganize it to create a new story or a poem or a book. <br/> I decided to start with the "Enough Python" exercise to understand how to learn python and then move on to "Corpus driven narrative generation" reorganize and recreate new narratives. 

First I will read in the string from a file using `open(...).read()`. I'll use a plain text file.<br/>
The following line will read my generated text `allTogether.txt` into a variable called `text`:


In [16]:
text = open('allTogether.txt').read()

In [17]:
text

'\nA couple of men standing next to each other.\nA crowd of people standing around each other.\nA group of people standing on top of a lush green field.\nA cell phone sitting on top of a table.\nA group of people sitting around a table.\nA group of men standing next to each other.\nA room filled with lots of furniture next to windows.\nA living room filled with furniture and a fire place.\nA woman is standing in front of a refrigerator.\nA little girl standing in front of a mirror.\nA car parked on the side of a road.\nA kitchen with a refrigerator and a stove.\nA room with a table , chairs and a television.\nA man standing in front of a window in a room.\nA display case filled with lots of different flavored donuts.\nA woman wearing a red hat and a red tie.\nA man in a suit and tie standing in a room.\nA couple of cars that are sitting in the snow.\nA statue of a man holding a banana.\nA view of a traffic light in a dark sky.\nA close up of a person holding a cell phone\na bowl of foo

This text has the "control character" in it, shown as \n. This is the new line symbol. <br/>
I want to get rid of this symbol so I will use the `print()` function. 

In [18]:
print(text)


A couple of men standing next to each other.
A crowd of people standing around each other.
A group of people standing on top of a lush green field.
A cell phone sitting on top of a table.
A group of people sitting around a table.
A group of men standing next to each other.
A room filled with lots of furniture next to windows.
A living room filled with furniture and a fire place.
A woman is standing in front of a refrigerator.
A little girl standing in front of a mirror.
A car parked on the side of a road.
A kitchen with a refrigerator and a stove.
A room with a table , chairs and a television.
A man standing in front of a window in a room.
A display case filled with lots of different flavored donuts.
A woman wearing a red hat and a red tie.
A man in a suit and tie standing in a room.
A couple of cars that are sitting in the snow.
A statue of a man holding a banana.
A view of a traffic light in a dark sky.
A close up of a person holding a cell phone
a bowl of food with a spoon in it.
a

Strings have a number of helpful "methods," which are little bits of code you can put after the variable name to return a copy of the string's text with some changes applied.<br/>
I will use the method `.split()`, which "splits" a string up into units. <br/>
To split a string up into lines:

In [19]:
text.split("\n")

['',
 'A couple of men standing next to each other.',
 'A crowd of people standing around each other.',
 'A group of people standing on top of a lush green field.',
 'A cell phone sitting on top of a table.',
 'A group of people sitting around a table.',
 'A group of men standing next to each other.',
 'A room filled with lots of furniture next to windows.',
 'A living room filled with furniture and a fire place.',
 'A woman is standing in front of a refrigerator.',
 'A little girl standing in front of a mirror.',
 'A car parked on the side of a road.',
 'A kitchen with a refrigerator and a stove.',
 'A room with a table , chairs and a television.',
 'A man standing in front of a window in a room.',
 'A display case filled with lots of different flavored donuts.',
 'A woman wearing a red hat and a red tie.',
 'A man in a suit and tie standing in a room.',
 'A couple of cars that are sitting in the snow.',
 'A statue of a man holding a banana.',
 'A view of a traffic light in a dark sky

To split the text into words:

In [20]:
text.split()

['A',
 'couple',
 'of',
 'men',
 'standing',
 'next',
 'to',
 'each',
 'other.',
 'A',
 'crowd',
 'of',
 'people',
 'standing',
 'around',
 'each',
 'other.',
 'A',
 'group',
 'of',
 'people',
 'standing',
 'on',
 'top',
 'of',
 'a',
 'lush',
 'green',
 'field.',
 'A',
 'cell',
 'phone',
 'sitting',
 'on',
 'top',
 'of',
 'a',
 'table.',
 'A',
 'group',
 'of',
 'people',
 'sitting',
 'around',
 'a',
 'table.',
 'A',
 'group',
 'of',
 'men',
 'standing',
 'next',
 'to',
 'each',
 'other.',
 'A',
 'room',
 'filled',
 'with',
 'lots',
 'of',
 'furniture',
 'next',
 'to',
 'windows.',
 'A',
 'living',
 'room',
 'filled',
 'with',
 'furniture',
 'and',
 'a',
 'fire',
 'place.',
 'A',
 'woman',
 'is',
 'standing',
 'in',
 'front',
 'of',
 'a',
 'refrigerator.',
 'A',
 'little',
 'girl',
 'standing',
 'in',
 'front',
 'of',
 'a',
 'mirror.',
 'A',
 'car',
 'parked',
 'on',
 'the',
 'side',
 'of',
 'a',
 'road.',
 'A',
 'kitchen',
 'with',
 'a',
 'refrigerator',
 'and',
 'a',
 'stove.',
 'A',


`.split()` produces a *list*. A list has multiple items in it—multiple strings. 
Assign a list to a variable like so:

In [21]:
words = text.split()

length of the text in words using the `len()` function:

In [22]:
len(words)

18289

Just like we discussed in class a function I like to think of alot is: RANDOMNESS + COMPUTATIONAL MODELS = ABILITY TO PRODUCE SUPRISES<br/>
So we will load the random function into a our python script.

In [23]:
import random

The `random.sample()` function takes two parameters: a list to draw samples from, and the sample you want to draw. (Samples are guaranteed not to be duplicates.)<br/>

To do something more than once, take the code you want to run more than once and tab it over once (i.e., indent each line by four spaces—you can do this in Jupyter Notebook by selecting the code and hitting `TAB`.) <br/>
Above the indented code, put `for i in range(n):` on a line, replacing `n` with the number of times you want to repeat. <br/>

### To create my computer generated novel I want to create a new narrative in the style of a poem from my previously generated text.<br/>   

1 - Splitting the text.<br/> 
I split my generated text of 18289 words into 800 words. (The cell after the next one shows how I do this and save them as txt files in my folder structure.) <br/> 

2 - Generating the poem.<br/> 
I want to write a loop that will take 800 words at a time and randomly select words from these 800 words in different ways. Then join all the words together in a certain structure to generate a poem. 

We start with 1 - Splitting the text. First line of code creates a text document with the title "1.txt"

In [11]:
f= open("1.txt","w+")

Second line of code, selects the words between 0 and 800 and joins them. Prints them. The writes them to the text document we saved above. 

In [12]:
selected = words[0:800]
joined = ' '.join(selected)
print(joined)
f.write(joined)

A couple of men standing next to each other. A crowd of people standing around each other. A group of people standing on top of a lush green field. A cell phone sitting on top of a table. A group of people sitting around a table. A group of men standing next to each other. A room filled with lots of furniture next to windows. A living room filled with furniture and a fire place. A woman is standing in front of a refrigerator. A little girl standing in front of a mirror. A car parked on the side of a road. A kitchen with a refrigerator and a stove. A room with a table , chairs and a television. A man standing in front of a window in a room. A display case filled with lots of different flavored donuts. A woman wearing a red hat and a red tie. A man in a suit and tie standing in a room. A couple of cars that are sitting in the snow. A statue of a man holding a banana. A view of a traffic light in a dark sky. A close up of a person holding a cell phone a bowl of food with a spoon in it. a 

3731

The third line of code, closes the document.

In [13]:
f.close() 

Now we move onto 2 - Generating the poem.<br/> The code below, selects and prints out 25 random selected words from the words 0 to 800 from my generated text.<br/> Then selects and prints out 10 randomly selected words from the words 0 to 800.<br/> Then joins them in a 6 word sentence structure.<br/> Then prints them.



In [10]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[0:800], 6)
        joined = ' '.join(selected)
        print(joined)

a a the a a clock
room a with building. a close
front and view river. their a
that filled a A a airplane
of holding A spoon a down
of of other. lots traveling donuts.
sign. side window skateboard of and
ear donuts. front down a beach.
table. A large man man of
man a over man table. on
a little top a man side
a in hand. a boat. a
A people room is ear a
clock standing a walking computer top
standing a sitting sitting holding holding
around a person toothbrush couple person
side top next other. large a
men food street. traffic of sandwich.
in a people standing red wall
piece standing pans. to white dark
A sandy a a wine on
little building. a on people man
to on large top train a
up front wedding around a a
tarmac. holding with up with on
woman with clock hand. people room
barn. water. men front lots a
field. a donuts. of a a
standing a sitting beach a a
teddy a man a of a
stop it. in statue a that
sign. of a red and church
holding scissors next a to walking
middle white to a a wine
on cak

I'm going to go through all the text snippets first. so words 800 to 1600 then words 2400 to 3200 and so on. Until we get to the end of the text which is made up of 18289.

In [14]:
f= open("2.txt","w+")

In [15]:
selected = words[800:1600]
joined = ' '.join(selected)
print(joined)
f.write(joined)

is doing a trick on a snowboard. a group of people standing next to each other. a couple of people riding bikes down a street. a group of people sitting around a table. a fire hydrant sitting in the middle of a forest. a fire hydrant sitting in the middle of a forest. a person is doing a trick on a skateboard. a man standing in front of a white surfboard. a man holding a pair of scissors in his hands. a group of people standing around a table. a book with a picture of a bird on it. a group of people standing next to an airplane. a close up of a person laying on a bed a clock that is on top of a table. a man standing in a kitchen preparing food. a woman sitting in front of a laptop computer. a view from the window of an airplane. a group of people riding skis on top of a snow covered slope. a close up of a mirror with a reflection of a person a white plate topped with a piece of cake. a man riding a snowboard down a snow covered slope. a man is standing in the snow on skis. a woman is s

3752

In [16]:
f.close() 

In [17]:
f= open("3.txt","w+")

In [18]:
selected = words[1600:2400]
joined = ' '.join(selected)
print(joined)
f.write(joined)

over a city with a mountain in the background. a black and white photo of a pair of scissors. a living room filled with furniture and a fire place. a building that has a sign on it a sign that says on the side of a building. a kitchen with a lot of pots and pans a display case filled with lots of different cakes. a kitchen with a refrigerator and a stove a group of people standing in a room. a close up of a person holding a cell phone a close up of a window with a red light a group of people standing in a room. a group of people standing around a luggage carousel. a tv sitting on top of a wooden table. a kitchen with a sink and a stove a kitchen with a sink , stove and refrigerator. a fire hydrant sitting on the side of a building. a clock that is on the side of a building. a close up of a keyboard and a mouse a bathroom with a tub , sink , and mirror. a bathroom with a sink and a mirror a close up of a person holding a cell phone a kitchen counter with a sink and a microwave a group o

3858

In [19]:
f.close() 

In [20]:
f= open("4.txt","w+")

In [21]:
selected = words[2400:3200]
joined = ' '.join(selected)
print(joined)
f.write(joined)

black and white photo of a plane flying in the sky. a person holding a pair of scissors in their hand. a laptop computer sitting on top of a wooden desk. a laptop computer sitting on top of a table. two people sitting at a table with laptops two people playing a video game in a living room. a person holding a pair of scissors in their hands. a group of people in a room watching a television. a close up of a person holding a baseball bat a group of people standing in front of a tv. a close up of a person holding a cell phone a group of people standing in front of a tv. a white refrigerator freezer sitting inside of a kitchen. a person cutting a cake with a knife. a pair of scissors sitting on top of a wooden table. a pair of scissors sitting on top of a wooden table. a person cutting a piece of cake with a knife. a couple of cups that are on a table a lit up christmas tree in a room. a birthday cake with candles on top of it. a red and white fire hydrant sitting on the side of a road. a

3730

In [22]:
f.close() 

In [70]:
f= open("5.txt","w+")

In [71]:
selected = words[3200:4000]
joined = ' '.join(selected)
print(joined)
f.write(joined)

ceiling. a large building with a clock tower on top. a large building with a train on it. a traffic light suspended over a city street. a vase with flowers in it on a table. a display of teddy bears in a store. a view of a city street at night. a statue of a bear sitting on top of a table. a clock on the side of a building. a man standing in front of a table filled with food. a woman holding a vase filled with flowers. a car driving down a street next to tall buildings. a woman wearing a hat and a hat. a close up of a remote control on a table a couple of men standing next to each other. a woman holding a cell phone in her hand. a man is flying a kite in the sky a group of people riding motorcycles down a street. a large cathedral with a clock on the wall. a dog laying on a bed next to a book. a pair of scissors sitting on top of a wooden table. a display case filled with lots of different flavored donuts. a group of people walking down a street. a black and white photo of a person hol

3667

In [72]:
f.close() 

In [73]:
f= open("6.txt","w+")

In [74]:
selected = words[4000:4800]
joined = ' '.join(selected)
print(joined)
f.write(joined)

vase with flowers in it a large white building with a large window. a black and white image of a vase with flowers in it. a white and red bus parked in front of a building. a bunch of different colored ties on a table. a person cutting a piece of paper with scissors. a large building with a large clock tower. a row of wooden benches sitting next to each other. a close up of a snow covered ground a close up of an orange on a plate a large umbrella sitting on top of a wooden table. a close up of a red and white clock. a picture of a stop sign in the dark. a street sign with a building in the background a stop sign that is on top of a pole. a close up of a vandalized stop sign a sign that says. a man sitting on a bench in front of a building. a sign on the side of a building. a row of urinals mounted to a wall. a black and white photo of a pair of scissors. a stop sign with graffiti written on it a close up of a pair of scissors on a table a man sitting on a bench under an umbrella. a sig

3679

In [75]:
f.close() 

In [76]:
f= open("7.txt","w+")

In [77]:
selected = words[4800:5600]
joined = ' '.join(selected)
print(joined)
f.write(joined)

of different types of scissors on a table. a row of urinals mounted to the side of a wall. a group of people standing next to each other. a group of people standing in a room. a kitchen with a stove and a sink a table topped with plates of food and drinks. a red umbrella sitting on top of a wooden table. a red fire hydrant in front of a building. a group of luggage sitting on top of a conveyor belt. a group of people walking down a street. a group of people standing in front of a building. a laptop computer sitting on top of a desk. a red stop sign sitting on the side of a road. a dog sitting in a chair in a room. a large building with a clock on it a living room filled with furniture and a fire place. a building with a sign on the side of it a room with a bed , table and chairs a blue and white clock on a wall a clock on a wall with graffiti on it. a tennis racket and a ball on a tennis court. a man standing in front of a mirror holding a camera. a train station with a train passing b

3724

In [78]:
f.close() 

In [79]:
f= open("8.txt","w+")

In [80]:
selected = words[5600:6400]
joined = ' '.join(selected)
print(joined)
f.write(joined)

up the side of a ramp. a large body of water with a sky background a pair of scissors sitting on top of a table. a pile of luggage sitting on top of a wooden floor. a room with a bed , a chair and a lamp. a living room with a couch and a table a kitchen with a sink , refrigerator , stove , and cabinets. a living room with a couch and a table a bed with a wooden frame and a wooden frame. a white refrigerator freezer sitting inside of a kitchen. a bathroom with a sink and a mirror a bathroom with a toilet and a shower. a bathroom with a shower , sink and mirror. a bathroom with a sink and a mirror a laptop computer sitting on top of a desk. a kitchen with a stove and oven in it a train station with a train on the tracks. a large building with a clock on the side of it. a sign that says on the side of a building. a red stop sign sitting on the side of a building. a stop sign in front of a large building. a close up of a pair of scissors on a table a red stop sign with a sticker on it. a g

3580

In [81]:
f.close() 

In [82]:
f= open("9.txt","w+")

In [83]:
selected = words[6400:7200]
joined = ' '.join(selected)
print(joined)
f.write(joined)

on it. a close up of a street sign with a sky background a room with a bed , desk , chair and television. a sign that reads. a building with a clock on the top of it a bunch of different colored vases sitting on a table. a view of a building with a clock on it. a red stop sign sitting on the side of a road. a red stop sign sitting on the side of a road. a statue of a man holding a baseball bat. a close up of a stop sign with a sky background a collage of photos showing a person holding a skateboard a man holding a tennis racquet on top of a tennis court. a close up of a person holding a toothbrush a close up of a person holding a toothbrush a close up of a nintendo wii remote a group of people standing in front of a tv. a man taking a picture of himself in a mirror. a room that has a bed and a chair in it a woman standing in front of a mirror holding a toothbrush. a red umbrella hanging from the ceiling of a building. a close up of a pair of scissors on a table a close up of a clock on

3614

In [84]:
f.close() 

In [85]:
f= open("10.txt","w+")

In [86]:
selected = words[7200:8000]
joined = ' '.join(selected)
print(joined)
f.write(joined)

of a wooden table. a clock on the side of a building. a blue and white bus driving down a street. a close up of a clock on a building a close up of a street sign with a sky background a clock on the side of a building. a train station with a train passing by. a picture of a person s reflection in a mirror. a pair of scissors sitting on top of a table. a close up of a stop sign with a sky background a pair of scissors sitting on top of a table. a white vase filled with lots of different colored flowers. a couple of white boats sitting on top of a field. a table that has a bunch of umbrellas on it a group of birds sitting on top of a sandy beach. a picture of a very nice looking room. a bunch of colorful umbrellas hanging from a ceiling. a row of colorful vases sitting on top of a table. a close up of a white and blue clock a close up of a red and white polka dot umbrella. a red and white vase sitting on top of a table. a red and white polka dot umbrella in a room. a close up of a clock 

3613

In [87]:
f.close() 

In [88]:
f= open("11.txt","w+")

In [89]:
selected = words[8000:8800]
joined = ' '.join(selected)
print(joined)
f.write(joined)

traffic light with a sky background a picture of a stop sign in the dark. a traffic light sitting on the side of a road. a black and white photo of a stop light. a close up of a stop sign with a sky background a bunch of red and yellow umbrellas hanging from a ceiling. a sign that says on the side of a building. a group of people standing outside of a building. a close up of a vase with flowers in it a pair of scissors sitting on top of a table. a black and white image of a clock on a wall. a person on a surfboard in the water. a stop sign with a street sign above it. a street sign with a building in the background a sign that says on the side of a building. a sign that is on the side of a building. a picture of a person holding a kite. a blue and white bird is flying in the sky a man is cutting a cake with a knife. a close up of a pair of scissors on a table a close up of a pair of scissors on a table a couple of street signs hanging from a pole. a large building with a clock on top o

3599

In [90]:
f.close() 

In [91]:
f= open("12.txt","w+")

In [92]:
selected = words[8800:9600]
joined = ' '.join(selected)
print(joined)
f.write(joined)

bed and a large window. a stop sign with graffiti written on it a group of people standing next to each other. a stop sign with a sticker on it. a close up of a vase with flowers in it a window that has a clock on it. a person is doing a trick on a skateboard. a close up of a clock on a wall a sign with a clock on it in front of a building. a sign that says. a blue and white vase sitting on top of a wooden floor. a picture of a person holding a skateboard. a flock of birds flying over a large body of water. a blue and white cake sitting on top of a table. a train on a track near a building a close up of a clock on a building a close up of a knife on a cutting board a close up of a clock on a building a close up of a white and black plate a pair of scissors sitting on top of a table. a man riding a snowboard down a snow covered slope. a cat sitting on a window sill next to a window. a close up of a bunch of green plants a man standing in front of a building. a man is taking a picture of

3684

In [93]:
f.close() 

In [94]:
f= open("13.txt","w+")

In [95]:
selected = words[9600:10400]
joined = ' '.join(selected)
print(joined)
f.write(joined)

black and white cat a man sitting on a surfboard in the water. a collage of photos with a clock on the wall a beach with a blue umbrella on top of it. a large building with a clock on top of it. a flock of birds sitting on top of a sandy beach. two people on a beach flying a kite a group of people standing on top of a sandy beach. a large white and blue vase on a table. a large body of water next to a bridge. a group of people standing on top of a sandy beach. a room with a bed and a desk with a lamp on it a close up of a pair of scissors on a table a black and white photo of a black and white bird. a room with a bed and a window a bedroom with a bed and a large window. a close up of a person holding an apple a close up of a person holding a kite a black and white photo of a bird flying in the air. a red and white cake with a red ribbon. a close up of a red and white umbrella a blue and white polka dot umbrella on a table. a group of vases sitting on top of a table. a black and white p

3583

In [96]:
f.close() 

In [97]:
f= open("14.txt","w+")

In [98]:
selected = words[10400:11200]
joined = ' '.join(selected)
print(joined)
f.write(joined)

up of a street sign with a sky background a blue and white decorated cake on a table. a man standing in a field with a frisbee. a sign that says on the side of a building. a sign that says on the side of a building. a black and white photo of a man on a skateboard. a black and white photo of a person holding an umbrella. a man in a suit and tie standing in a room. a large building with a clock on top of it. a city street filled with lots of traffic. a group of people standing on top of a bridge. a black and white photo of a bathroom sink. a black and white photo of a man holding a skateboard. a herd of sheep grazing on a lush green field. a couple of giraffes that are standing in the grass. a large group of sheep in a field. a bunch of umbrellas that are in the grass. a bunch of vases that are on a wall. a crowd of people standing around a large crowd of people. a view of a river with a bridge in the background. a bunch of vases that are sitting on the ground. a view of a park with a l

3690

In [99]:
f.close() 

In [100]:
f= open("15.txt","w+")

In [101]:
selected = words[11200:12000]
joined = ' '.join(selected)
print(joined)
f.write(joined)

field. a large building with a clock on the top. a bunch of green and yellow flowers in a tree. a bunch of green and yellow flowers in a tree. a bed with a wooden frame and a wooden headboard. a statue of a man standing next to a clock. a train traveling through a lush green forest. a close up of a person wearing a suit and tie a close up of a mirror with a reflection of a dog a large white bird sitting on top of a wooden table. a white bird sitting on top of a window sill. a statue of a bear in the middle of a forest. an umbrella sitting on top of a wooden table. a close up of a small mouse on a desk a group of people sitting on top of a bench. a person is standing on a rock with a surfboard. a person is standing on a rock with a surfboard. a black and white photo of a man in a suit. a close up of a pair of scissors on a table a blue and white kite flying in the sky. a black and white photo of a man wearing a tie. a black and white photo of a man wearing a tie. a close up of a person 

3548

In [102]:
f.close() 

In [103]:
f= open("16.txt","w+")

In [104]:
selected = words[12000:12800]
joined = ' '.join(selected)
print(joined)
f.write(joined)

a close up of a clock on a wall a view of the water from the sun. a large building with a clock on it a large clock hanging from the ceiling of a building. a close up of a person holding a toothbrush a close up of a person holding an open umbrella a close up of a pair of scissors on a table a picture of a cake on a table. a close up of a person holding a book a bed with a black and white blanket on it. a woman sitting on a bed in a room. a bunch of kites that are in the sky. a close up of a white and blue frisbee. a close up of a nintendo wii remote a close up of a nintendo wii remote a close up of a clock on a building a black and white photo of a baseball game. a couple of white sinks in a room. a pair of scissors sitting on top of a table. a group of birds sitting on top of a roof. a room filled with lots of wooden furniture. a bunch of books that are on a table. a bunch of ties hanging from a wall. a group of people walking down a street. a man standing in front of a store front. a

3676

In [105]:
f.close() 

In [106]:
f= open("17.txt","w+")

In [107]:
selected = words[12800:13600]
joined = ' '.join(selected)
print(joined)
f.write(joined)

of a bear in the water. a large building with a clock on the side of it. a black bear standing in the middle of a forest. a pair of scissors sitting on top of a table. a close up of a large body of water a person holding an umbrella in the rain. a black and white photo of a train. a wooden bench sitting in front of a window. a pink flower in a vase on a table. a bench sitting in the middle of a lush green field. a close up of a person holding a pair of scissors a bunch of kites that are in the sky. a black and white photo of a bed a city street filled with lots of traffic. a person in the water with a surfboard a black and white photo of a clock. a tree with a bunch of fruit on it a group of people standing on top of a snow covered field. a wooden bench sitting in front of a window. a close up of a mirror in a room a room with a large bed and a large window. a close up of a bird in a tree a train on a train track with a sky background a red and white train on a train track a picture of

3690

In [108]:
f.close() 

In [109]:
f= open("18.txt","w+")

In [110]:
selected = words[13600:14400]
joined = ' '.join(selected)
print(joined)
f.write(joined)

of a man with a tie. a close up of a pair of scissors on a table a man with a hat and a tie. a black and white photo of a clock on a wall. a pair of scissors sitting on top of a table. a black and white photo of a person holding an umbrella a bunch of colorful kites hanging from a ceiling. a close up of a pair of scissors on a table a blue umbrella sitting on top of a lush green field. a bunch of umbrellas hanging from a ceiling. a bunch of kites that are flying in the sky. a close up of a person holding a nintendo wii controller a close up of a person holding a cell phone a bunch of ties hanging from a wall. a bunch of different colored ties hanging on a wall. a picture of a person holding a kite. a close up of a person wearing a costume a close up of a red and white clock. a group of colorful umbrellas sitting on top of a table. a bunch of umbrellas that are in the air. a bunch of different colored ties hanging on a wall. a colorful umbrella is hanging on a wall a large building with

3701

In [111]:
f.close() 

In [112]:
f= open("19.txt","w+")

In [113]:
selected = words[14400:15200]
joined = ' '.join(selected)
print(joined)
f.write(joined)

a large clock tower with a sky background a display case filled with lots of different flavored donuts. a person sitting on a bench under a tree a collage of photos with a clock on the wall a kite flying in the air with a sky background a view of a city skyline from a distance. a red and white fire hydrant sitting in the middle of a forest. a group of birds sitting on top of a window sill. a close up of a stop sign with a sky background a pair of scissors sitting on top of a table. a group of men standing next to each other. a man and a woman standing next to each other. a collage of photos with a stop sign a group of people standing in front of a building. a box of donuts sitting on top of a table. a bathroom with a shower and a toilet. a large bridge spanning the width of a river. a small bird standing on top of a cage. a group of people walking down a street. a group of people standing on a tennis court. a crowd of people standing around a train station. a large building with a cloc

3701

In [114]:
f.close() 

In [115]:
f= open("20.txt","w+")

In [116]:
selected = words[15200:16000]
joined = ' '.join(selected)
print(joined)
f.write(joined)

building with a large clock mounted to it s side. a window that has a vase with flowers in it. a woman standing in a living room next to a couch. a view of a hallway from a hallway. a group of men standing next to each other. a bathroom with a sink and a mirror a white toilet sitting next to a white sink. a group of men standing next to each other. a black and white photo of a bedroom a close up of a person holding a surf board a bunch of different colored scissors on a table. a large building with a clock on it. a man in a black shirt and black pants holding a white frisbee. a man in a suit and tie standing in a room. a bunch of kites that are in the air. a woman in a black shirt and a black tie a picture of a cake on a table. a person is holding a pair of scissors. a group of women standing next to each other. a black and white photo of a vase with flowers. a bride and groom cutting their wedding cake. a person jumping a skate board in the air a group of people standing next to each 

3697

In [117]:
f.close() 

In [118]:
f= open("21.txt","w+")

In [119]:
selected = words[16000:16800]
joined = ' '.join(selected)
print(joined)
f.write(joined)

on a table. a couple of vases sitting on top of a table. a close up of a mirror on a wall a pair of scissors sitting on top of a table. a close up of a person holding a cell phone a blue and white striped umbrella sitting on top of a table. a person holding an umbrella in front of a building. a mirror with a reflection of a man in it. a black and white photo of a man doing a trick on a skateboard. a group of people standing next to each other. a glass of wine sitting on top of a table. a group of people standing next to each other. a pair of scissors sitting on top of a table. a black and white photo of a clock on a wall. a birthday cake with a train on it a black and white photo of a woman holding a knife a close up of a person wearing a tie a group of people standing on top of a bridge. a close up of a clock on a table a white and blue boat sitting on top of a beach. a close up of a person holding a doughnut a clock mounted to the side of a wall. a traffic light sitting on the side o

3747

In [120]:
f.close() 

In [121]:
f= open("22.txt","w+")

In [122]:
selected = words[16800:17600]
joined = ' '.join(selected)
print(joined)
f.write(joined)

on a skateboard on a ramp. a group of people sitting on top of a bench. a close up of a person wearing a suit and tie a black and white photo of a toaster oven. a plane flying in the sky over a mountain. a close up of a clock on a wall a pair of scissors sitting on top of a wooden table. a close up of a clock on a wall a young boy holding a nintendo wii game controller. a close up of a vase with flowers in it a close up of a pair of scissors in a car a clock on a wall in a room. a kitchen with a stove , sink , and cabinets. a plate of food with a knife and fork. a close up of a blender on a counter a cup of coffee sitting on top of a table. a close up of a pair of scissors on a table a kitchen with a refrigerator and a stove a close up of a pair of scissors on a table a little girl standing in front of a table. a bunch of toilets that are in a room. a close up of a blender on a table a table with a plate of food and a cup of coffee. a wooden bench sitting on top of a wooden bench. a ki

3592

In [123]:
f.close() 

In [124]:
f= open("23.txt","w+")

In [125]:
selected = words[17600:18400]
joined = ' '.join(selected)
print(joined)
f.write(joined)

a pile of ripe bananas sitting on top of a table. a pair of scissors and a pair of scissors. a black and white photo of a fire hydrant. a pair of scissors sitting on top of a table. a black and white photo of a wooden bench. a statue of a man holding a kite. a bunch of colorful kites hanging from a string. a black and white photo of a wooden bench a group of people standing next to each other. a boat floating on top of a body of water. a black and white photo of a baseball player. a man in a suit and tie holding a cell phone. a group of people standing around a large elephant. a train traveling over a bridge over a river. a man riding a skateboard down the side of a ramp. a black and white photo of a pair of scissors. a group of people flying kites in a park. a tall building with a clock on top of it. a black and white photo of a pair of scissors. a woman in a red shirt is talking on a cell phone. a group of black and white zebras in a field. a man in a suit and tie standing in a room.

3149

In [126]:
f.close() 

Now we move onto 2 - Generating the poem parts again.<br/> The code below keeps generating new poems from the text snippets we have just created. So we used the words 800 to 1600, then 2400 to 3200 and so on.

In [138]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[800:1600], 6)
        joined = ' '.join(selected)
        print(joined)

crowd a a laptop next a
of top on front bowl sitting
a black cupcakes bird on of
of on kitchen a of large
the group close night. close around
of the window bunch scissors up
a around holding surfboard man of
close fire pair at man a
large a a white up sign.
pair shelf sidewalk riding snow a
wearing lush a hanging green snow
a a of hat it it
sign. is a down a clock.
large up a airplane in of
a of scissors a top table.
on umbrella top brick a of
people a table. couch. window a
hydrant to chairs up an hydrant
riding a vase with of a
of up sitting a a snowboard
of crowd of green pair on
skis standing of standing a surfboard.
in a a court. a standing
building. a on in person up
court. a a bird large table.
in of riding of a of
a and filled on sitting his
a a a field. couple on
white standing a court. computer. group
a around snow a people slope.
on laying room. of ledge a
a a vase clock in sitting
on of black a dark of
people a a vase to a
to a a slope. cake. pair
hat sitting vases in diffe

In [139]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[1600:2400], 6)
        joined = ' '.join(selected)
        print(joined)

display man from a person a
a room. of building. a a
street. street. furniture filled of a
of a a a row man
other. fire in a of a
a a top a and beach.
woman a a a filled people
mirror room person of building. fire
with a standing field a next
a light skateboard pile trees. of
front truck the keyboard a woman
a room. bathroom a group man
people a a picture standing a
a a table. a and a
cell hydrant a at freezer cell
on white a colorful of sitting
on scissors. a a hydrant standing
and a a a his stove
a mouse side down holding standing
field clock with window snowboard. inside
kite holding a close next pair
a in a is of is
table of cell a kitchen group
of on in a in bathroom
front standing pans truck a a
a a colorful a riding a
sitting people of a a of
group a a lot a a
a a with a top a
next in top from taking up
that in a a holding a
with a skateboard group a kitchen
a a a with , sky
of of men a a sitting
man counter a group a a
other. refrigerator. pair a a white
sitting a riding side s

In [142]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[2400:3200], 7)
        joined = ' '.join(selected)
        print(joined)

a living and on a a a
of bathroom laptop a red with table
into row a the a in a
a a with with traffic a on
laptop. people game a cake wall. cutting
in trees. a a a a next
a walking and a person a person
a with group of a candles of
of on a a a a in
a a a a sitting pair in
sitting sitting close a close a tv.
table of a candles of a watching
desk and computer a vases a person
range up a clock. middle a cups
person knife. walking woman a a with
luggage a side man of skis street.
a that person a to person bat
a a close sky. people an the
to walking a a on a on
in bunch a sitting on white other.
on a lots close on the a
and a and with people pair of
clock brush holding traffic on a pizza.
street. standing of topped vases. with group
black a and a lit person of
a table. in people man of person
a group a desk. a a on
group on a of a a sitting
ground. up bunch a cake holding a
desk group a on laptop sitting that
of of and are on that with
doing a next with of to of
a a on person trick sitting 

In [145]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[3200:4000], 6)
        joined = ' '.join(selected)
        print(joined)

forest. on a woman filled a
of a herd with a photo
a a city a with snow
a display wearing a building wooden
a on it. of a the
table clock a and wall and
city a a holding with from
of and a white vases with
a holding forest. side up people
wearing on a a phone in
in of front vase cell in
a a and black bed next
a phone a a photo on
of on a and close each
a white standing in a a
a a a men flowers. in
on woman clock at street a
phone view a to of building
a a white a room. man
of a a air. that around
building clock phone walking colored men
sitting taking a black a man
a of a bed on each
with down wearing tall bed a
view a track. flowers up of
the down range standing with a
on front dark person large table.
a of of a clock of
a close a a black and
hand. of of food. a of
a white of close building people
a close flowers of up from
up a laying a a in
black in street. around dark the
store. covered a umbrella control on
a holding a herself a umbrella.
teddy ceiling. photo a display. surfboard


In [146]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[4000:4800], 4)
        joined = ' '.join(selected)
        print(joined)

up a side bathroom
on a on of
a of that standing
on and wall. of
on a toilet a
a on food with
to it. a picture
mirror place. of a
a on of with
table people on a
it. table a red
clock of sink urinals
front a standing middle
cutting top a a
a game close table
a close bathroom a
a a on a
with of up building.
on a that snow
clock people sign and
sign white skateboard. on
, of red a
a a on a
a a in a
a picture a each
of of of stop
that and of reads
furniture a sign of
side a ground stop
a parked clock toilet
a of pair it
a each up background
bunch with a a
of a sign a
living a pair scissors
a up a other.
a a parked top
in sink front ties
sink of a it.
side of standing pair
with person standing side
up and sign side
of says front up
close front man of
tower a side on
paper and it. the
and bathroom red a
snow food and a
side bathroom with other.
row close a sitting
place. top to wooden
front and of up
is a is large
people middle a the
up is middle a
a building on sign
pair close next a
that c

In [147]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[4800:5600], 6)
        joined = ' '.join(selected)
        print(joined)

wall of bed green mountain an
view with wall. furniture sign clock
a stop photo road. a a
sink table of and standing toilet
the of window. black on sitting
background range a surf a urinals
on a standing a airplane stop
a traffic clock up room close
sitting standing building. a blue top
sitting a of of white standing
road. a white clock furniture clock
a side a of a of
colored of sign red it surf
with clock a plates in a
mountain range side front place. on
blue on the a of with
a umbrella. of by. a side
to pair with a tie. a
of of a window. up a
photo on train room in red
ball a with a apple street.
of person in a of a
sink the and a of board
other. a of on a wearing
up from with the the of
sitting of a sitting umbrella. lot
stop with in of a from
the kitchen a tie. of a
a riding on a the building.
sky room fire hanging a close
an a filled a and close
a front light a middle passing
that of and sitting in mounted
clock a side mounted side a
blanket to room. holding passing person
in bui

In [149]:
for i in range(10):
    for y in range(25):
        selected = random.sample(words[5600:6400], 6)
        joined = ' '.join(selected)
        print(joined)

a a hanging and of written
living ties a a , hanging
of sink with of sign chair
over a has water a a
, of top a sink up
wall. a background a of a
a of white blue luggage a
oven a kitchen. tower. with a
man window. a shirt a building
sitting picture road. and a a
a clock and a of are
over scissors a a a on
of a wall. and close flying
with and toilet and on a
on close on a mirror pair
of clock of bicycle on stop
, sign lots tree. with the
parked and with with cake a
lots close building table of pair
living sign on a wooden of
that photo station a clock up
a clock sign side clock a
standing on suit towering a on
and a a a wooden from
wall. on and background on a
close a a and of tennis
background frame. sink up of with
a of refrigerator that wall red
of road. body top scissors a
a that , and close white
a building. a wall has clock
on stop of a window. the
a with up clock and a
of sign plane sign a luggage
up of a a a bunch
sitting a white with in a
furniture. with a sign in woman
a red f

In [153]:
for i in range(20):
    for y in range(15):
        selected = random.sample(words[6400:7200], 5)
        joined = ' '.join(selected)
        print(joined)

scissors. a a from holding
sitting a other. a top
clock street a man kites
of stop a close pair
a with a that red
through a on a field.
sign covered picture in a
water. on that it. a
large building. a on a
dark a a a group
large table people sitting view
kites on a in top
a a it. it umbrella
baseball a statue train of
in of a red of
in from with with court
on of large with that
a clock it. a close
holding the forest. mounted in
track. street clock large each
a standing building close up
in a with pair vase
a in next black sitting
that train in skateboard light
of wall people a a
in a a in in
group of person clock bed
a a table. standing open
a train scissors a a
a side a of red
in a colored snow park.
are bedroom pair close large
umbrella skateboard of park. in
stop of reads. up on
of red a holding that
red in a a the
of a track. building scissors
with on skate on photos
covered of window close person
ceiling trees photos dark. says
of a clock with street
street. a a green clock
of it 

In [155]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[7200:8000], 6)
        joined = ' '.join(selected)
        print(joined)

of tree. of flowers. a a
close of bed with man that
of top a a from black
clock surf table dot and a
a close a top a black
a of building. close a scissors
lush a a sitting traffic a
a a a sitting with surfboard.
street with building on a a
air. a teddy a tooth a
a water street bird tree. holding
the a bird a lined traffic
a wii bench traffic a sky
of sitting group lush kite on
a in park. on in sky
sitting a body a table. table
the dot white a colorful a
of pair the a a wall
colorful and pole different a train
of of a on branch. wall
a close top of people close
that umbrella a table. a traffic
a side in on water close
of striped a and polka clock
that with clock holding street a
clock on a a a street
of with red a the of
of up field. a train on
traffic of red a a from
a in light a train clock
comforter top sitting a of of
close a the background is person
a cake of a a a
blue of on field. a a
a with a down sitting a
of on a up picture picture
pair a background on a park.
building man chr

In [156]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[8000:8800], 6)
        joined = ' '.join(selected)
        print(joined)

on a top of a down
a elephant sky person a of
side bird laptop. skateboard her on
fire black in stop a holding
of a yellow of building. that
side of of a a a
a a bunch tennis on black
side building. a vase a of
on pair scissors a a water.
next up close building. sitting up
up to the on a a
on white pair sign pair and
with a on on table. top
red a a on city. photos
of table. her a mirror. people
close court. tennis a road. sitting
down it. of scissors light a
it. a hanging on on of
on in top a over a
a pair side of close of
of tracks a a sign in
holding up people up the window
white window each other. of a
a man a a pair a
yellow a man scissors light. a
being from a close a a
photo snow. picture mirror. white on
wooden scissors sitting of a scissors
building white close table top up
up is top sky large a
each a up the picture front
man up a a the photos
on side white a bird of
a standing black of a a
on in a other. next traffic
on large pair kite. wooden table
a standing side background

In [157]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[8800:9600], 6)
        joined = ' '.join(selected)
        print(joined)

background group skateboard. front suit a
a on a street black with
on a a on suit it
a a a and of a
next holding in a of holding
close flock clock a next man
of sill in up wall a
up a picture of of scissors
a a night. book. wall. a
has of holding of to of
controller. standing doing side other. store.
for a a flowers kitchen man
on a next passing of skateboard.
flat mirror a on pair sitting
a other. a train a the
of in mounted a in a
the each of with a up
on on vase it sitting wii
sticker with book. woman picture. in
the of guitar. court. picture cell
a wii stuffed bench. room of
on a a flying toilet. a
a to tie. up up on
a that a wii a knife
next with street holding close night.
with up a mirror building a
controller holding with passing station man
a a a it. a a
pair man window passing a of
top. of a a front a
stop clock and a holding with
of close with of close on
and man clock a of cell
clock vases a a a to
a building. a boat a up
a on on cell sitting clock
of holding a of each man


In [158]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[9600:10400], 6)
        joined = ' '.join(selected)
        print(joined)

a on a a in white
a room of next flowers. of
close on a a of a
of up group on up of
a of a woman on a
teddy table sidewalk beach white nintendo
on on a beach a front
in and a a a front
a red bedroom front black a
a holding a standing picture white
holding on holding hanging in a
holding a an in a a
man on a a a and
a bedroom row a close an
a hanging sitting it. black a
up a window filled front in
room up of and tie person
close to on rain. blue are
a with it close of holding
on white table. surfboard. woman a
man on wave woman suit each
tower photo the of racket up
a a in photo man with
up on umbrella kite a a
is with hanging person up group
a man of with a a
top it. sitting room a pile
it. black a collage a a
beach. suit television a close a
a a and a a wall.
on people and a photo standing
mirror of up of a a
a on food up a picture
with close front window. with man
a suit white and of a
scissors of and scissors luggage. window.
close a a of with each
on close vase a wall people
bathro

In [160]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[10400:11200], 5)
        joined = ' '.join(selected)
        print(joined)

window a window. of laptop
umbrella. close a the in
woman black and on white
with small a a umbrella
street. a of with a
of standing small black grass.
a in window. top of
street. of are birds close
of in umbrella pair close
white of and a window.
up sheep of a a
hat. a knife of a
image room. pink on blue
and clock. beach. with tie
a are white a bridge
that white dress window. of
group an down sitting in
a a a a sheep
of front man holding red
with stuffed a skis of
red bunch phone. view the
that of a room. standing
standing a of red photo
and black a a scissors
photo a a red with
and view a in in
of kitchen on , street.
on and a a black
an large a in up
a a tie blue on
a a large photo park
up background. white rail. sheep
of of a large an
a a person bathroom white
of the a a next
person photo white red light
animals large of mountain sitting
white to wooden in next
a a pair of small
background. in person side frisbee.
a phone. up and knife
and a a black in
white a the standing standing

In [162]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[11200:12000], 5)
        joined = ' '.join(selected)
        print(joined)

furniture and and yellow cake
riding flying walking each on
the night a tie the
close in close a laptop
up flowers a traffic of
of close large of mirror
image close tree. up group
a man table a toothbrush
at park. standing a riding
person under holding the a
it. man a frame and
a large a holding top
black close the tree. up
kite and the of with
a clock and ocean bench.
photo a side a of
a and up a a
scissors blue mirror a building
a room a a a
group his up bunch man
a it. of a holding
with clock a of a
sitting people a tree and
black of in green up
a a a filled suit
a holding a of clock.
and at table. statue wall.
sitting man ocean sitting of
woman black of brushing tie.
scissors table. on room forest.
a night are a with
city of to the tree.
pair a a a a
mirror pair a close bunch
lots sitting on bathroom of
at mirror. a street. on
top up up of of
table. a frame black cell
of a on the night.
a white of table cell
people scissors walking up clock
microwave up clock table of
sitting sill.

In [163]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[12000:12800], 6)
        joined = ' '.join(selected)
        print(joined)

a of a street of with
kites on person close close pair
birds close clock a clock sitting
a close it photo a a
types mirror close a scissors up
pair person a sitting table. side
of a scissors a a standing
with a middle pair down table.
clock a holding blue window a
in of it group on photo
down of close up scissors a
surfboard. of of top surf sitting
holding wall. close a holding a
sitting a umbrellas. on tree playing
an and sitting pair on person
a a of on around bed
on large of on clock a
of of a of a a
in walking from a river. a
remote wave down close white of
street wearing close close of pair
a wearing ceiling of game of
water on red a on of
front striped a a in on
pair remote group a of around
a on a a the of
building on up a of a
table group of of of a
sky it collage on person of
down close of and a of
to of close of a a
that bunch a scissors in a
a lots of bird a up
of pair wall of a clock
mirror filled in and in person
of room tree train a a
of group a holding from bunch
a white

In [164]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[12800:13600], 6)
        joined = ' '.join(selected)
        print(joined)

up a lush and kitchen. of
in on a colorful sitting a
kites a a inside sitting each
grass. wooden a different a on
people table. in on colorful with
person traffic. the types in covered
the of close the table. white
wooden a of a wall. pair
urinals of a close in a
middle bunch a of a in
mounted sandy of man beach. a
and a around umbrellas a group
close of of sitting a mirror.
bathroom top on top a close
with white urinals birthday sign a
a a collage close different a
picture a bunch a blue stop
is pink black bear and field.
of of of of of a
refrigerator of vase television black pair
of of wall. top side a
vase in wall. kite to bench
blue up a on a a
in in a and in street.
the tower. room skateboard sign and
sitting a of bench. standing of
pair in each a next sitting
top different standing around a a
wall. sandy wall of a ties
on of a wall. a bunch
with bear train. and large lot
white of train street white and
middle clock of in mirror. on
of table. mounted on a tree
lit sitting wooden f

In [166]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[13600:14400], 5)
        joined = ' '.join(selected)
        print(joined)

of of sitting a sitting
a the blue with and
a kites flying it green
the wall. the of flying
a holding sitting of person
are colored birds tie. a
of stop with a on
a holding chairs wall. of
a a flying black scissors
up through on sand. of
a the a a of
a umbrella person white a
with skateboard. of a sign
a a of from top
food picture table wooden of
of and and on are
a a table a air.
on in hydrant close table
with a an picture colorful
person , close top and
white colorful with photo and
beach close sink hydrant bathroom
in sitting through of with
black a on chairs wall.
black room a person a
on a a on on
a a photo of and
up a a of building
and a bunch top a
close bathroom a table with
tree. scissors. on colorful black
holding christmas and that a
picture riding a small has
a top kite. a a
up from the phone the
white of of white a
a white of an umbrella
person and air. sign a
on water. a clock sky
of hydrant of group white
of on is building racket.
a wave a clock on
sitting a a of picture

In [167]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[14400:15200], 5)
        joined = ' '.join(selected)
        print(joined)

close large bunch red building.
on group up on of
wall. photo of standing close
on background a on standing
close people a middle close
a colorful air. in up
close and bedroom and a
a to people group a
window. of collage around sitting
of in a wooden a
holding window road. under sky
a in book. and a
around a stop lots window.
phone a close pair table.
up on clock a top
in pair sitting a stop
of a collage a a
a photos clock a an
a a pair a clock
standing up of of collage
a of a bathroom up
a of table wooden large
vandalized in case of person
of scissors table a book.
hydrant large a close width
under a of close the
the with sitting station. building.
close a a a of
standing a man on a
building. of of holding an
of on large of hydrant
stop a a wall. man
the white of table. a
a table. table a a
a standing top a a
bunch around group of a
a a of tennis city
air. with wall the a
a air standing kites top
a black umbrella crowd of
with umbrellas pair stop people
sign up up bunch a
in bed wall.

In [168]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[15200:16000], 4)
        joined = ' '.join(selected)
        print(joined)

bride a flower clock
standing a down a
a a a of
close toilet black sitting
sandy group a sitting
sky hands. umbrella. table
flock over large people
a in beach a
view a of a
different on sitting a
a a white building
of a view surfboard.
a in phone a
man an standing blue
scissors in standing of
and a sitting a
in woman of a
person ties window a
with umbrella. a a
window black riding a
to table standing a
each train and a
close a s of
on a a his
street. is in of
a bunch a cake
broccoli pair of phone
black umbrella on it
clock a a a
close woman of on
car. beach. table. a
on room. holding a
a with it beach.
table. on riding on
a view in a
close the a men
scissors. a standing with
an colorful pair air.
a light man picture
sky a a and
kite. white a group
window jetliner standing a
a a hanging close
building a on group
group ceiling. woman a
walking bench. pair room.
from in of man
street in of a
view close top a
in a a a
holding board night. flower
woman a a the
clock a close building
a a on 

In [169]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[16000:16800], 6)
        joined = ' '.join(selected)
        print(joined)

window. up a side trick place.
to a person of group of
filled of of of the man
of skateboard a on other. of
a with a of and and
of in white table. to up
wine building cell trick bathroom top
person kite. the brick a people
countryside. a bathroom. a knife a
a mountain. playing the painting and
a an traffic a furniture of
lush a a a mounted a
chairs of with white of a
boat up a place. large a
top in on and up doing
umbrella over a in flying a
pair a a building fire close
kites a in from air hydrant
each on standing in people of
holding a with a a tennis
space. pair of a a standing
with a person a man group
skateboard on in the people a
sitting a flowers on of with
a in furniture sitting wall with
of a on a a knife
a clock of sitting of of
the clock up of black on
person a and on a of
traffic a table. table. that toilet.
of white on black a jumping
top scissors front jumping a a
and filled of a it. traffic
sitting phone a clock person a
of a of with red middle
of kites a a a of
of umbrel

In [170]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[16800:17600], 6)
        joined = ' '.join(selected)
        print(joined)

a up clock board view a
fire a in blender up a
fire of ground up is hydrant
stop standing a forest. in background
plane a body a of on
of forest. up of wall wall
red young a photo forest. a
bench a a bench a middle
group body of in hydrant. bench
a a mountain. of snowy red
a forest. cake of close close
of in of a bench. the
a hanging in pair on and
fire cup a table. a of
the a of of wooden to
a on view flowers. view a
coffee street a knife each a
a close of a plate sun.
a on and a a on
of bed a fire on jacket.
standing a person cabinets. it a
surfboard a a person a wall
wooden shower. a kite. white hydrant
a a of of a a
a dark. up a kite. a
sitting and of up in sitting
with on table is close a
photo of table in pair a
of a phone that a purple
up game sitting toilet table. of
middle on of stove over toaster
controller. the metal from of in
of up at scissors person up
people a bathroom different a on
and middle on of flying on
on clock. to view a sink
a a of riding close vases
from it th

In [15]:
for i in range(25):
    for y in range(10):
        selected = random.sample(words[17600:18400], 4)
        joined = ' '.join(selected)
        print(joined)

NameError: name 'random' is not defined